integration-tests/src/test/java/com/pip/chatbot/controller/tests/karateAuthorization.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Zrobiłbym raczej plik z konfiguracją(karateConfiguration.json). Tam mielibyśmy adminUser, adminPassword i baseUrl. Autoryzowalibyśmy się tak: https://stackoverflow.com/questions/51244771/how-to-implement-basic-auth-using-karate